src/сomponents/Experimental/FirstType/FirstType.test.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import Adapter from 'enzyme-adapter-react-16';
+import { shallow, configure } from 'enzyme';
+import { shallowToJson } from 'enzyme-to-json';
+import FirstType from './FirstType';
+
+configure({adapter: new Adapter()});
+
+describe('FirstType', () => {
+  it('should render correctly', () => {
+    const output = shallow(
+      <FirstType>First title</FirstType>
+    );
+    expect(shallowToJson(output)).toMatchSnapshot();
```

Comment:
You could add `enzyme-to-json/serializer` to jest config. Then you will no need to write `shallowToJson` everywhere

Reply:
done